rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/service/LRAService.java

```diff
@@ -173,7 +173,8 @@ public void finished(Transaction transaction, boolean fromHierarchy) {
             // the LRA is top level or it's a nested LRA that was closed by a
             // parent LRA (ie when fromHierarchy is true) then it's okay to forget about the LRA
 
-            if (transaction.afterLRANotification()) {
+            if (!transaction.hasPendingActions()) {
+                // this call is only required to clean up cached LRAs. Follow up with a JBTM to avoid this
```

Comment:
Does the JBTM exist yet? It would be useful to reference the actual JBTM in the comment

Reply:
I raised [JBTM-3250](https://issues.redhat.com/browse/JBTM-3250) and reworded the comment.